src/menu/settingsMenu.ts

```diff
@@ -37,6 +37,10 @@ export const settingsMenu: MenuItemConstructorOptions = {
       ...separator,
       visible: !IS_MAC,
     },
+    {
+      label: "Refresh",
+      role: "forceReload",
+    },
```

Comment:
This should probably not be part of the setting menu. If we go with this action type i would prefer to see it in the window menu. It will also need to match the key binds of the already existing refresh, Thanks for the contribution.